haystack/utils/augment_squad.py

```diff
@@ -179,7 +180,7 @@ def augment(
     word_possibilities: int = 20,
     replace_probability: float = 0.4,
     batch_size: int = 16,
-    device: str = "cpu:0",
+    device: torch.device = torch.device("cpu:0"),
```

Comment:
`augment()` is called by `augment_squad()` and it passes the device as a str so this won't work as intended.

Reply:
same for `load_glove()`